### PR TITLE
[BACKLOG-27754] Hadoop Cluster UX - Create New Unsecured Cluster Configuration from Site XML

### PR DESCRIPTION
**Pull request by e-cuellar**

### CODE REVIEW

File: kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    try {
+      if ( nc.getConfigId() != null ) {
+        delNamedCluster( metaStore, nc );
+      }
+      String newClusterId = generateNewClusterId( null );
+      nc.setConfigId( newClusterId );
+      addConfigProperties( nc );
+      saveNamedCluster( metaStore, nc );
+    } catch ( Exception e ) {
+      /*commonDialogFactory.createErrorDialog( spoon.getShell(),
```
Comment (peterrinehart) [inline kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java]: Can these comments be deleted, or are they there for documentation purposes?

File: kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+
+    return nc.getName();
+  }
+
+  private void saveNamedCluster( IMetaStore metaStore, NamedCluster namedCluster ) {
+    try {
+      namedClusterService.create( namedCluster, metaStore );
+    } catch ( MetaStoreException e ) {
+      /*commonDialogFactory.createErrorDialog( spoon.getShell(),
```
Comment (peterrinehart) [inline kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java]: Can these comments be deleted, or are they there for documentation purposes?

File: kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+          String path = getNamedClusterConfigsRootDir( xmlMetaStore ) + "/" +  namedCluster.getConfigId();
+          try {
+            FileUtils.deleteDirectory( new File( path ) );
+          } catch ( IOException e ) {
+            // Do nothing. The config directory will be orphaned but functionality will not be impacted.
+          }
+        }
+      }
+    } catch ( MetaStoreException e ) {
+      /*commonDialogFactory.createErrorDialog( spoon.getShell(),
```
Comment (peterrinehart) [inline kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java]: Can these comments be deleted, or are they there for documentation purposes? Should probably do something with this exception, write it to console output if not a log file.

File: kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java (hunk 1 above)
Comment (e-cuellar) [reply kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java]: Peter. I left those comments as a reference for now. That's what the current code based on the SWT UI does. I will know more about what to do with those exceptions as James progresses on the UI front.

File: kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java (hunk 2 above)
Comment (e-cuellar) [reply kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java]: Peter. I left those comments as a reference for now. That's what the current code based on the SWT UI does. I will know more about what to do with those exceptions as James progresses on the UI front.

File: kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java (hunk 3 above)
Comment (e-cuellar) [reply kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/MultishimNamedCluster.java]: Peter. I left those comments as a reference for now. That's what the current code based on the SWT UI does. I will know more about what to do with those exceptions as James progresses on the UI front.

**Pull request merged**